README.md

```diff
@@ -27,7 +27,7 @@ the environment of the integration tests.
 
 ## Installation
 
-Pennyworth is tested on [openSUSE 13.1](http://en.opensuse.org/Portal:13.1).
+Pennyworth is tested on [openSUSE 13.2](https://en.opensuse.org/Portal:13.2) and [SLES12].
```

Comment:
It's confusion that it says `tested on 13.2` and below the installation instructions are for 13.1.